java/java.source.base/src/org/netbeans/api/java/source/ElementUtilities.java

```diff
@@ -365,86 +370,97 @@ public Iterable<? extends Element> getMembers(TypeMirror type, ElementAcceptor a
         
         return result;
     }
-    
-    private static final ElementAcceptor ALL_ACCEPTOR = new ElementAcceptor() {
-        @Override
-        public boolean accept(Element e, TypeMirror type) {
-            return true;
+
+    private static void addAlways(Element element, List<Element> list, Map<String, List<Element>> members) {
+        String name = element.getSimpleName().toString();
+        List<Element> namedMembers = members.get(name);
+        if (namedMembers == null) {
+            namedMembers = new ArrayList<>();
+            members.put(name, namedMembers);
+        }
+        if (list != null) {
+            list.add(element);
         }
-    };
+        namedMembers.add(element);
```

Comment:
nitpick - lines 376-380+384 could be replaced with:

```java
members.computeIfAbsent(name, (e) -> new ArrayList<>()).add(element);
```

Reply:
@matthiasblaesing thanks for taking a look. It is quite difficult to get reviewers in summer :)

good idea. Will try to get this in. This would improve readability I agree.